src/priya/SumOfNumbersConvertUpperToLowercase.java

```diff
@@ -0,0 +1,43 @@
+//Program 2: Do Sum of all numbers and Convert Uppercase into lowercase, lowercase into uppercase using ASCII.
+//Given String = "Te1ch8n9oC6reDi5TS1"
+//Expected String = "30tECHNOcREdIts"
+
+package priya;
+
+public class SumOfNumbersConvertUpperToLowercase {
+
+	int sum = 0;
+	String s = "";
+	char ch;
+	int total;
+
+	int sumOfNumbers(String word) {
+		for (int index = 0; index < word.length(); index++) {
```

Comment:
this can be handle in below method, I hope you just want to make separate due to sum operation